packages/react-components/react-table/stories/Table/Virtualized.stories.tsx

```diff
@@ -0,0 +1,236 @@
+      newItem.file.label = `${newItem.file.label}-${i}-${index}`;
+      newItem.lastUpdate.label = `${newItem.lastUpdate.label}-${i}-${index}`;
+      newItem.lastUpdated.label = `${newItem.lastUpdated.label}-${i}-${index}`;
+      contentList.push(newItem);
+    });
+  }
+  return contentList;
+};
+
+const fullItemList: Item[] = generateContent();
+
+// Simple wrapper class to control scroll view height
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    overflowAnchor: 'none',
+    overflowY: 'auto',
+    width: '100%',
+    height: '100%',
+    maxHeight: '750px',
+  },
+});
+
+export const Virtualized = () => {
+  const columns: ColumnDefinition<Item>[] = React.useMemo(
+    () => [
+      createColumn<Item>({
+        columnId: 'file',
+        compare: (a, b) => {
+          return a.file.label.localeCompare(b.file.label);
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'author',
+        compare: (a, b) => {
+          return a.author.label.localeCompare(b.author.label);
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdated',
+        compare: (a, b) => {
+          return a.lastUpdated.timestamp - b.lastUpdated.timestamp;
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdate',
+        compare: (a, b) => {
+          return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
+        },
+      }),
+    ],
+    [],
+  );
+  const {
+    getRows,
+    sort: { getSortDirection, toggleColumnSort, sort },
+  } = useTableFeatures(
+    {
+      columns,
+      items: fullItemList,
+    },
+    [useTableSort({ defaultSortState: { sortColumn: 'file', sortDirection: 'ascending' } })],
+  );
+
+  const headerSortProps = (columnId: ColumnId) => ({
+    onClick: (e: React.MouseEvent) => {
+      toggleColumnSort(e, columnId);
+    },
+    sortDirection: getSortDirection(columnId),
+  });
+
+  const rows = sort(getRows());
+  const styles = useStyles();
+
+  const rowFunc = React.useCallback(
+    (index: number) => {
+      const item: Item = rows[index].item;
+      return (
+        <TableRow key={item.file.label}>
+          <TableCell>
+            <TableCellLayout media={item.file.icon}>{item.file.label}</TableCellLayout>
+          </TableCell>
+          <TableCell>
+            <TableCellLayout
+              media={<Avatar name={item.author.label} badge={{ status: item.author.status as PresenceBadgeStatus }} />}
+            >
+              {item.author.label}
+            </TableCellLayout>
+          </TableCell>
+          <TableCell>{item.lastUpdated.label}</TableCell>
+          <TableCell>
+            <TableCellLayout media={item.lastUpdate.icon}>{item.lastUpdate.label}</TableCellLayout>
+          </TableCell>
+        </TableRow>
+      );
+    },
+    [rows],
+  );
+
+  return (
+    <div className={styles.container}>
+      <Table sortable>
```

Comment:
```suggestion
      <Table sortable noNativeElements>
```

I thought that there was some perf bottleneck with using semantic table elements?

Reply:
We've made some significant perf improvements since then so this may be resolved (in perf testing latest build it wasn't causing any concerns) - will do some comparison and reverification when we put this demo up as a separate PR.